File: stepxx_lambda_destination/example01_lambda_destination_with_different_destinations/lambda/destined.ts
exports.handler = async (event: any) => {
    console.log('Received event:', event.Records[0].Sns);
    const {Success} = JSON.parse(event.Records[0].Sns.Message);
    console.log(Success)
    if (Success) {
        console.log("Success");
        return {
            success: true,
            Message: "Hello World from SNS"
        };
    }
    console.log("Failure");
    throw new Error("Failure from event, Success = false, I am failing!");

};

File: step26_lambda_destination/example01_lambda_destination_with_different_destinations/Python/example01_lambda_destination_with_different_destinations/lambda/destined.py
import json


def handler(event):
    print('Received Event: {}'.format(json.dumps(event['Records'][0]['Sns'])))
    Success = event['Records'][0]['Sns']['Message']
    print(Success)
    if Success:
        print("Success")
        return {
            'success': True,
            'Message': "Hello World from SNS"
        }
    print('Failure')
